addons/libs/config.lua

```diff
@@ -142,114 +142,116 @@ function merge(t, t_merge, path)
     for key in pairs(t) do
         keys[tostring(key):lower()] = key
     end
+    
+    if t_merge then
```

Comment:
This diff is awful to read due to the indentation. Was all you changed the if check? If so, I'd prefer for you to just use this:
```lua
    if not t_merge then
        return t
    end
```